src/mono/mono/mini/debugger-protocol.h

```diff
@@ -0,0 +1,374 @@
+#ifndef __MONO_DEBUGGER_PROTOCOL_H__
```

Comment:
Is the same protocol going to be reused by the coredbg code ?

Reply:
Yes.

Reply:
If you want to take a look here is the icordebug implementation: https://github.com/dotnet/runtime/compare/master...thaystg:thays_implement_icordbg